redaxo/src/core/boot.php

```diff
@@ -15,7 +15,7 @@
     throw new Exception('PHP version >=' . REX_MIN_PHP_VERSION . ' needed!');
 }
 
-foreach (array('HTDOCS_PATH', 'BACKEND_FOLDER', 'REDAXO') as $key) {
+foreach (['HTDOCS_PATH', 'BACKEND_FOLDER', 'REDAXO'] as $key) {
```

Comment:
Die Array-Schreibweise war glaube ich bewusst, damit in niedrigeren PHP-Versionen kein Parse Error kommt, sondern die Exception ganz oben in der boot.php greift.
Deswegen wird die Datei hier auch ausgeschlossen: 
https://github.com/redaxo/redaxo/blob/master/.php_cs.dist#L23-L25

Reply:
hm ok.. beim lokalen fixing ist das hier so geändert worden... müsste man wohl mal testen warum das mit der aktuellsten version nicht richtig funktioniert..

Reply:
Damit die Config greift, ist glaube ich wichtig dass dein working directory der root des repos ist.
Oder die Config-Datei explizit angeben.
Bei mir wird die Datei nicht gefixt, wenn ich den Fixer über das ganze Projekt laufen lasse.

Reply:
Oder vielleicht wegen Windows?

Reply:
stimmt.. https://github.com/redaxo/redaxo/pull/1728